javascript/index.html

```diff
@@ -76,7 +76,7 @@
 } else {
   // This code could also be modified to ask for access to the user's genomics data too
   // You just need one additional scope: 'https://www.googleapis.com/auth/genomics'
-  // And an additional library: {name: 'genomics', version: 'v1beta'}
+  // And an additional library: {name: 'genomics', version: 'v1'}
```

Comment:
Alternatively, this comment could be replaced with one that directs the reader to api-client-javascript if they want to add in the genomics client.